.github/workflows/app-tests.yaml

```diff
@@ -54,33 +54,29 @@ jobs:
           run: php .github/build-packages.php
 
         # We always install PHP deps because of the UX Translator, which requires `var/translations` to exists
-        - name: Install App dependencies
+        - name: Install PHP dependencies
           uses: ramsey/composer-install@v3
           with:
             working-directory: test_apps/encore-app
             dependency-versions: highest
 
-        - if: matrix.ux-packages-source == 'php-vendor'
-          name: Refresh dependencies from vendor/
-          working-directory: test_apps/encore-app
-          run: yarn
-          env:
-            YARN_ENABLE_HARDENED_MODE: 0
-            YARN_ENABLE_IMMUTABLE_INSTALLS: 0
+        - working-directory: test_apps/encore-app
+          run: pnpm install --ignore-workspace
```

Comment:
`--ignore-workspace` allows to ignore the `pnpm-workspace.yaml` from parent folders, and install dependencies from `test_apps/encore-app` only (`test_apps/encore-app` is not in the workspace)